### ISSUE TITLE
loadgen: need a basic load generator to test machine performance

### ISSUE THREAD

**Issue by rjnn** [cockroachdb/loadgen#107]
We need a "diagnostic" load generator. Occasionally customers come to us and say "I ran query X" and even though we can analyze their schema if they give it to us, we also want to get a quick understanding of their machine's performance characteristics (since they don't always share their data) before going forward.

This is currently easy if we are primarily interested in their machine's raw read or write QPS (as we can point them to `ycsb` or `kv`), but for some other questions, there isn't a straightforward solution. For instance, if we want to understand their scan performance, asking them to go through the burden of running `dbgen` and `tpc-h` load is a lot of work.

We ideally want a load generator that does the following tasks, but also kills two birds with one stone by providing a blueprint for simple tasks that we can point users to that does:

1. Bulk inserts optimized with `RETURNING NOTHING` syntax.
2. `IMPORT` data from CSV.
3. Simple large scans (`COUNT(*)`).
4. Repeated single key 100% reads (i.e. the simplest degenerate case of `kv`)

Any other simple tasks that we would want to know to diagnose very basic performance characteristics @petermattis?

**Comment by petermattis** [comment]
This seems similar to https://github.com/cockroachdb/cockroach/issues/17369. My suspicion is we'd want to run these diagnostics at a lower-level than SQL and to have this be built directly into cockroach.

**Comment by rjnn** [comment]
I agree, but why not both? It also seems much easier to do this version using our existing load generator codebase.

**Comment by petermattis** [comment]
No strong reason not to do both, though I'd like to see #17639 happen first as I think that will be easier for users to use and potentially something we can run automatically.
